api/test/integration/user.e2e-spec.ts

```diff
@@ -604,4 +614,162 @@ describe('User Controller Tests', () => {
+        phoneNumberVerified: true,
+      }),
+    });
+
+    const jurisdiction = await prisma.jurisdictions.create({
+      data: {
+        name: 'single_use_code_1',
+        allowSingleUseCodeLogin: true,
+        rentalAssistanceDefault: 'test',
+      },
+    });
+    emailService.sendSingleUseCode = jest.fn();
+
+    const res = await request(app.getHttpServer())
+      .post('/user/request-single-use-code')
+      .send({
+        email: storedUser.email,
+      } as RequestMfaCode)
+      .set({ jurisdictionname: jurisdiction.name })
+      .expect(201);
+
+    expect(res.body).toEqual({ success: true });
+
+    expect(emailService.sendSingleUseCode).toHaveBeenCalled();
+
+    const user = await prisma.userAccounts.findUnique({
+      where: {
+        id: storedUser.id,
+      },
+    });
+
+    expect(user.singleUseCode).not.toBeNull();
+    expect(user.singleUseCodeUpdatedAt).not.toBeNull();
+  });
+
+  it('should request single use code, but jurisdiction does not allow', async () => {
+    const storedUser = await prisma.userAccounts.create({
+      data: await userFactory({
+        roles: { isAdmin: true },
+        mfaEnabled: true,
+        confirmedAt: new Date(),
+        phoneNumber: '111-111-1111',
+        phoneNumberVerified: true,
+      }),
+    });
+
+    const jurisdiction = await prisma.jurisdictions.create({
+      data: {
+        name: 'single_use_code_2',
+        allowSingleUseCodeLogin: false,
+        rentalAssistanceDefault: 'test',
+      },
+    });
+    emailService.sendSingleUseCode = jest.fn();
+
+    const res = await request(app.getHttpServer())
+      .post('/user/request-single-use-code')
+      .send({
+        email: storedUser.email,
+      } as RequestMfaCode)
+      .set({ jurisdictionname: jurisdiction.name })
+      .expect(400);
+
+    expect(res.body.message).toEqual(
+      'Single use code login is not setup for single_use_code_2',
+    );
+
+    expect(emailService.sendSingleUseCode).not.toHaveBeenCalled();
+
+    const user = await prisma.userAccounts.findUnique({
+      where: {
+        id: storedUser.id,
+      },
+    });
+
+    expect(user.singleUseCode).toBeNull();
+  });
+
+  it('should request single use code, but user does not exist', async () => {
+    const jurisdiction = await prisma.jurisdictions.create({
+      data: {
+        name: 'single_use_code_3',
+        allowSingleUseCodeLogin: true,
+        rentalAssistanceDefault: 'test',
+      },
+    });
+    emailService.sendSingleUseCode = jest.fn();
+
+    const res = await request(app.getHttpServer())
+      .post('/user/request-single-use-code')
+      .send({
+        email: 'thisEmailDoesNotExist@exygy.com',
+      } as RequestMfaCode)
+      .set({ jurisdictionname: jurisdiction.name })
+      .expect(201);
+    expect(res.body.success).toEqual(true);
+
+    expect(emailService.sendSingleUseCode).not.toHaveBeenCalled();
+  });
+
+  it('should login successfully through single use code', async () => {
```

Comment:
this test should have stayed in the auth.e2e.spec since its still hitting`/auth/loginViaSingleUseCode`